### ISSUE TITLE
Entities highlighted in front-end

### ISSUE THREAD

**Issue by ziodave** [insideout10/wordlift-plugin#461]
The following entity appears as highlighted:

![image](https://cloud.githubusercontent.com/assets/11438/22333893/f980b690-e3e0-11e6-9916-ba1bb3943395.png)

This is due to the following markup in the front-end:

```
<strong><span id="urn:enhancement-b8dd6be3-30f6-e388-ccea-1f70b2c1c391" class="textannotation disambiguated wl-thing" itemid="http://open.salzburgerland.com/de/entity/barrierefrei">Barrierefreies</span> SalzburgerLand</strong>
```

The entity is actually not selected in the backend. A page preview in the backend doesn't actually show the entity. Resaving the page still shows the markup (the preview doesn't).

**Comment by ziodave** [comment]
I was able to reproduce it:
 1. create a post,
 2. run the analysis and select some entities,
 3. delete one of the entities from the vocabulary.

The entity will appear as _disambiguated_ in front-end.

**Comment by ziodave** [comment]
Merged in 3.9.7.
